applications/rag/metadata.yaml

```diff
@@ -46,7 +46,7 @@ spec:
         defaultValue: ""
       - name: cluster_name
         varType: string
-        required: true
+        defaultValue: "ai-on-gke"
```

Comment:
While you're at it, can you also update kubernetes_namespace to default to `ai-on-gke` to match the default value used for Ray and Jupyter